### ISSUE TITLE
Windows' last working version 12.30, anything beyond this (both pre-release/release) results in "spawn C:\Windows\system32\cmd.exe ENOENT"

### ISSUE THREAD

**Issue by Whipplash** [ansible/vscode-ansible#681]
### Summary

Opening after doing more testing. Switched to VSCode from Vim recently for teaching folks internally, but upon using the extension for the language server, on both my home system as well as my work system, I'm seeing the following 2x messages result from the extension:

spawn C:\Windows\system32\cmd.exe ENOENT

Ansible-lint is not available. Kindly check the path or disable validation using ansible-lint

Upon more testing, I've found the last working version to be 12.30. 12.57 is the first release where I see both of these messages pop up. Moving from 12.30 to 12.57 it's not clear in the documentation as-is, but is a remote extension + WSL a necessity from here onward? Just looking for some information to get this working at work. Thanks!

### Extension version

1.0.90

### VS Code version

1.73.0

### Ansible Version

```shell
➜  ~ ansible --version
ansible [core 2.12.10]
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/mjk/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  ansible collection location = /home/mjk/.ansible/collections:/usr/share/ansible/collections
  executable location = /bin/ansible
  python version = 3.8.10 (default, Jun 22 2022, 20:18:18) [GCC 9.4.0]
  jinja version = 2.10.1
  libyaml = True
➜  ~
```

### OS / Environment

Windows 10. WSL (Ubuntu), no remote extensions.

**Comment by priyamsahoo** [comment]
To use the extension in Windows, WSL is a necessity. The failure message needs to be improved for unsupported platforms and we are working on it. Thanks.

**Comment by slamanna212** [comment]
I am running this extension with ansible installed via WSL and still get these errors.

**Comment by marko-hologram** [comment]
> I am running this extension with ansible installed via WSL and still get these errors.

It's not just WSL that's a requirement. I had this error, but then I realized that I'm opening a folder inside WSL Ubuntu from my Windows VS Code instance, not using any remote extensions.

I fixed this by installing Remote Development extension (https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack) and actually opening a folder inside WSL Ubuntu via remote access. Then I opened command palette and found command "WSL: Reopen folder in WSL" which opened that folder inside VS Code instance that is properly aware of WSL. Another options is just connecting to WSL distro from VS Code and browse to whatever folder.

I looked at that error message a few times and then looking at the path "C:\Windows\system32\cmd.exe" I realized that it's trying to open something in my Windows filesystem, not my WSL Ubuntu instance. Then I realized I'm probably just missing these extensions.
